fimsauthz-api/fims-authz.yaml

```diff
@@ -274,7 +274,7 @@ components:
         nbf: 1509470774
         iss: fims.arc.nasa.gov/fimsAuthServer
         jti: 4d4c1a63-0959-48cc-885a-3648fd582db7
-    JwsCompactSerialization:
+    xUtmMessageSignature:
```

Comment:
this Schema's description does not say what is it relative to the data structure in the data exchange.